packages/restapi/src/lib/chat/helpers/inbox.ts

```diff
@@ -52,7 +52,12 @@ export const getInboxLists = async (
         toDID: ''
       }
     }
-    feeds.push({ ...list, msg: message, groupInformation: list.groupInformation });
+
+    const messageResponse = await Promise.all([message])
+      .catch((e) => console.log(e.message))
```

Comment:
Don't add .catch since we already catch it at upper-level fn and want this to throw an error if something goes wrong.